Prefabs/Interactions/Controllables/SharedResources/Scripts/RotationalTransformDrive.cs

```diff
@@ -0,0 +1,108 @@
+﻿namespace VRTK.Prefabs.Interactions.Controllables
+{
+    using UnityEngine;
+    using Zinnia.Extension;
+    using Zinnia.Data.Type;
+    using Zinnia.Data.Attribute;
+    using Zinnia.Tracking.Velocity;
+    using Zinnia.Tracking.Follow.Modifier.Property.Rotation;
+    using VRTK.Prefabs.Interactions.Interactables;
+
+    /// <summary>
+    /// A rotational drive that directly manipulates a <see cref="Transform.rotation"/> to control the rotational angle without the use of any physics.
+    /// </summary>
+    public class RotationalTransformDrive : RotationalDrive
+    {
+        #region Reference Settings
+        /// <summary>
+        /// The <see cref="InteractableFacade"/> that controls the movement of the drive.
+        /// </summary>
+        [Tooltip("The InteractableFacade that controls the movement of the drive."), InternalSetting, SerializeField]
+        protected InteractableFacade interactable;
+        /// <summary>
+        /// The <see cref="GameObject"/> that contains the meshes for the control.
+        /// </summary>
+        [Tooltip("The GameObject that contains the meshes for the control."), InternalSetting, SerializeField]
+        protected GameObject interactableMesh;
+        /// <summary>
+        /// The <see cref="TransformPositionDifferenceRotation"/> to drive the rotation of the control.
+        /// </summary>
+        [Tooltip("The TransformPositionDifferenceRotation to drive the rotation of the control."), InternalSetting, SerializeField]
+        protected TransformPositionDifferenceRotation rotationModifier;
+        /// <summary>
+        /// The <see cref="ArtificialVelocityApplier"/> apply artificial angular velocity to the control after releasing.
```

Comment:
`to apply`

Reply:
the original sentence doesn't make sense anyway, I'm going to change it to:

`The ArtificialVelocityApplier that applies artificial angular velocity to the control after releasing.`

Reply:
👍